core/main.go

```diff
@@ -165,5 +165,9 @@ func main() {
 	if err := stopRunningServices(sdk); err != nil {
 		logrus.WithField("module", "main").Fatalln(err)
 	}
+
+	if err := c.Cleanup(); err != nil {
```

Comment:
This function is only accessible for `DockerContainer`, it could be good to have it for the `Container` interface. Not blocking at all

Reply:
Why `Container` should have that one? Container is used by sdk and sdk is not and it shouldn't be responsible for cleanups. That interface was created to manage containers, not the container "space" like networks, swarm etc..